remote.go

```diff
@@ -126,7 +126,12 @@ func (e *Error) Error() string {
 }
 
 func (wd *remoteWD) execute(method, url string, data []byte) ([]byte, error) {
-	debugLog("-> %s %s\n%s", method, url, data)
+	loggedurl, err := logURL(url)
```

Comment:
If the function signature is changed as suggested above, inline this call in the debugLog function call.

Otherwise, call this variable loggedURL, as per https://github.com/golang/go/wiki/CodeReviewComments#initialisms